### ISSUE TITLE
Hugo添加音乐最终章之悬浮音乐播放器 | code show的博客

### ISSUE THREAD

**Issue by CorPython** [CorPython/CorPython.github.io#60]
https://corpython.github.io/post/hugo%E6%B7%BB%E5%8A%A0%E9%9F%B3%E4%B9%90%E6%9C%80%E7%BB%88%E7%AB%A0%E4%B9%8B%E6%82%AC%E6%B5%AE%E9%9F%B3%E4%B9%90%E6%92%AD%E6%94%BE%E5%99%A8/ 

 Hugo添加音乐最终章之悬浮音乐播放器

**Comment by Chow-Ray** [comment]
这个歌曲的链接，是外链对吗？像前一篇，用直接的歌曲链接不好使呗？

**Comment by CorPython** [comment]
> @Chow-Ray
> 这个歌曲的链接，是外链对吗？像前一篇，用直接的歌曲链接不好使呗？
是的,我之前是将歌曲都存在我的服务器上,用七牛云获取的外链,但是最近过期了没续费,所以歌曲都失效了,还有部分图片也加载不出来了
